src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,28 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    userId,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      { 'TodoInfo--completed': completed },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user && <UserInfo user={user} key={userId} />}
+    </article>
```

Comment:
Don't forget about code style
```suggestion
    <article className={classNames('TodoInfo', {
       'TodoInfo--completed': completed 
     })}
    >
      <h2 className="TodoInfo__title">
        {title}
      </h2>
      
      {user && <UserInfo user={user} key={userId} />}
    </article>
```